rules-tests/Naming/Rector/Class_/RenamePropertyToMatchTypeRector/Fixture/change_bare_property.php.inc

```diff
@@ -1,21 +0,0 @@
-<?php
-
-namespace Rector\Tests\Naming\Rector\Class_\RenamePropertyToMatchTypeRector\Fixture;
-
-final class ChangeBareProperty
-{
-    private ?\PDO $_pdo;
```

Comment:
removing fixture should be avoided, instead modify (show as skipped) to see how behaviour just changed